# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/pagination/fieldset-00E-ref.html
# Commit: 2684fdcd6fbf
# Full Hash: 2684fdcd6fbff5abe54a3e1c018e7188abb4076c
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-11-26 09:34:48
# Regressor Bug: 471015
# File Overlap Count: 1
# Description:
#   Bug 471015 - [css-break] Implement <fieldset> fragmentation.  r=TYLin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D53709
# ==============================================================================

diff -r e29eecc5147c -r 2684fdcd6fbf layout/reftests/pagination/fieldset-00E-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/reftests/pagination/fieldset-00E-ref.html	Mon Nov 25 21:45:22 2019 +0000
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html class="reftest-paged">
+<head>
+  <title>Reference: fieldset fragmentation</title>
+  <link rel="author" title="Mats Palmgren" href="https://bugzilla.mozilla.org/show_bug.cgi?id=471015">
+  <meta name="flags" content="paged">
+  <style>
+@page { size:5in 3in; margin:0.5in; }
+html,body {
+    color:black; background-color:white; font:0.5in/1 monospace; padding:0; margin:0;
+}
+fieldset {
+  padding: 0;
+  margin: 0;
+  border: 0.1in solid black;
+}
+legend {
+  padding: 0;
+  margin: 0;
+  width: 3em;
+  height: 2em;
+  box-sizing: border-box;
+  border: 0.1in solid black;
+}
+x {
+  display: block;
+  page-break-after: always;
+}
+</style>
+</head>
+<body>
+  <fieldset><legend></legend><x></x></fieldset>
+</body>
+</html>